src/cli/commands/__tests__/lint.test.ts

```diff
@@ -167,17 +170,46 @@ describe('lint', () => {
           })
         )
         .stdout()
-        .command(['lint', validCustomSpecPath, '-r', 'http://foo.local/ruleset.yaml'])
+        .command(['lint', validCustomOas3SpecPath, '-r', 'http://foo.local/ruleset.yaml'])
         .it('given valid remote ruleset file, outputs no issues', ctx => {
           expect(ctx.stdout).toContain('No errors or warnings found!');
         });
     });
+
+    describe('when a standard oas3 ruleset provided through option', () => {
+      test
+        .stdout()
+        .command(['lint', invalidOas3SpecPath, '-r', standardOas3RulesetPath])
+        .it('outputs warnings in default format', ctx => {
+          expect(ctx.stdout).toContain('Applying custom rules. Automatic rule detection is off.');
```

Comment:
I am seeing the word custom a lot and it feels funny. We do not want to give people the feeling that there is "normal" and "custom", there are just rulesets and rules. If you do not specify a ruleset, we will detect a default, but we don't need to keep saying "Woah there's a custom thing, this is pretty different!" 

Speccy would just tell you what rulesets it was trying to load and what rules it found, but didnt ever care if they were core or third party https://github.com/wework/speccy/blob/master/lib/loader.js#L133

Reply:
Sure, I'm open to suggestions. Does `Applying third-party rules` sound good?